azure-stack/aks-hci/set-proxy-settings.md

```diff
@@ -79,6 +79,38 @@ Set-AksHciConfig -proxySetting $proxySetting -...
 
 In order to exclude all addresses in a domain, you need to add the domain to the `noProxy` list. Use a leading period rather than a wildcard (*) character. In the sample, the addresses `.contoso.com` excludes addresses `prefix1.contoso.com`, `prefix2.contoso.com`, and so on.
 
+## Setting proxy for Azure Stack HCI and Windows Server clusters with machine wide proxy settings
+
+If you already have machine wide proxy settings on your Azure Stack HCI/Windows Server cluster, these settings may override any AKS specific proxy settings and lead to a failure while installation. 
+
+To detect if you have machine wide proxy settings, run the following script on **each** of your physical cluster nodes.
+
+```powershell
+$http_proxy = [System.Environment]::GetEnvironmentVariable("HTTP_PROXY", "Machine")
+$https_proxy = [System.Environment]::GetEnvironmentVariable("HTTPS_PROXY", "Machine")
+$no_proxy = [System.Environment]::GetEnvironmentVariable("NO_PROXY", "Machine")
+
+if ($http_proxy -or $https_proxy) {
+    if (-not $no_proxy) {
+        Write-Host "Problem Detected! A machine-wide proxy server is configured, but no proxy exclusions are configured"
+    }
+}
+```
+Configure machine-wide proxy exclusions on **each** of the physical cluster hosts where the problem was detected.
+
+> [!NOTE]
+> We recommend having the same proxy settings on all the nodes in the failover cluster. Having different proxy settings on different physical nodes in the failover cluster may lead to unexpected issues.
+
+Run the following PowerShell script and replace the $no_proxy parameter string with a suitable NO_PROXY exclusion string for your environment.
+
+```powershell
+$no_proxy = localhost,127.0.0.1,.svc,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
```

Comment:
By bad, I missed a quote on this string...also lets just reemphasize here that it's likely they will need to exempt their domain name (the prior section on exclusions mentions that).

Can we update this line to have the missing quote and the domain suffix example:

$no_proxy = "localhost,127.0.0.1,.svc,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,.contoso.com"